tests/models/test_marrmotm01.py

```diff
@@ -25,8 +26,19 @@ def forcing_file(self, sample_marrmot_forcing_file):
         return sample_marrmot_forcing_file
 
     @pytest.fixture
-    def model(self, forcing_file: Path, mocked_config):
-        m = MarrmotM01(version="2020.11", forcing=forcing_file)
+    def generate_forcing(self, forcing_file):
+        forcing = load_foreign('marrmot',
+                                directory=str(Path(forcing_file).parent),
+                                start_time='1989-01-01T00:00:00Z',
+                                end_time='1992-12-31T00:00:00Z',
+                                forcing_info={
+                                    'forcing_file': str(Path(forcing_file).name)
+                                })
```

Comment:
Indent is one space to many
```suggestion
                               directory=str(Path(forcing_file).parent),
                               start_time='1989-01-01T00:00:00Z',
                               end_time='1992-12-31T00:00:00Z',
                               forcing_info={
                                   'forcing_file': str(Path(forcing_file).name)
                               })
```